.github/workflows/copy-workflow.yml

```diff
@@ -63,105 +55,56 @@ jobs:
-        working-directory: ${{ env.TARGET_REPO_DIR }}
-        run: |
-          # We want our modules to support two Go versions at a time.
-          # As of August 2021, Go 1.17 is the latest stable.
-          # go.mod's Go version declares the language version being used.
-          # As such, it has to be the minimum of all Go versions supported.
-          # Bump this every six months, as new Go versions come out.
-          TARGET_VERSION=1.16
-
-          # Note that the "<" comparison doesn't understand semver,
-          # but it should be good enough for the foreseeable future.
-          CURRENT_VERSION=$(go list -m -f {{.GoVersion}})
-
-          if [[ $CURRENT_VERSION < $TARGET_VERSION ]]; then
-            echo "GO_VERSION_BUMP=1" >> $GITHUB_ENV
-
-            # Update the version in go.mod. This alone ensures there's a diff.
-            go mod edit -go $TARGET_VERSION
-
-            # In the future, "go fix" may make changes to Go code,
-            # such as to adapt to language changes or API deprecations.
-            # This is largely a no-op as of Go 1.17, and that's fine.
-            go fix ./...
-            git add .
-
-            # We don't tidy, because the next step does that.
-            # Separate commits also help with reviews.
-            git commit -m "bump go.mod to Go $TARGET_VERSION and run go fix"
-          fi
-    - name: go mod tidy (on initial deployment and on new Go version)
-      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 || env.GO_VERSION_BUMP == 1}}
-      uses: protocol/multiple-go-modules@v1.2
-      with:
-        working-directory: ${{ env.TARGET_REPO_DIR }}
-        run: |
-          go mod tidy
-          if ! git diff --quiet; then
-            git add .
-            git commit -m "run go mod tidy"
-          fi
-    - name: gofmt -s (on initial deployment and on new Go version)
-      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 || env.GO_VERSION_BUMP == 1}}
-      working-directory: ${{ env.TARGET_REPO_DIR }}
-      run: |
-        gofmt -s -w .
-        if ! git diff --quiet; then
-          git add .
-          git commit -m "run gofmt -s"
-        fi
-    - name: determine files to add
-      # By setting the environment variable, it's possible to programmatically add / modify this list.
-      # See https://github.com/protocol/.github/blob/38135c75e47839623bf9b2748275d8c6167a8fa8/.github/workflows/copy-workflow.yml#L163-L168 for an example, how we used to make use of this.
-      run: echo "FILES=${{ toJson(github.event.inputs.files) }}" >> $GITHUB_ENV
+    - name: Run steps specific to go
+      if: matrix.cfg.deploy_go
+      uses: ./template-repo/.github/actions/copy-workflow-go
+    - name: Run steps specific to versioning
+      if: matrix.cfg.deploy_versioning
+      uses: ./template-repo/.github/actions/copy-workflow-versioning
     - name: Add files
       run: |
-        for f in $(jq -r ".[]" <<< ${{ toJson(env.FILES) }}); do
+        for f in $(jq -r '.[]' <<< $FILES); do
```

Comment:
Just for the log output it would be nicer if we could continue using `env.FILES` (then GH Actions does the replacement before running the script).

Reply:
I restored the version that renders the values in the output.